src/MasterSite.php

```diff
@@ -327,6 +327,22 @@ function (): void {
             10
         );
 
+        // Fix WPML-related RTL issue.
+        $remove_rtl_fix = function (): void {
+            global $sitepress;
+            // This RTL fix does not seem a good idea.
+            // Probably it was a bad attempt at solving the issues `url_to_postid` creates.
+            remove_action('wp_head', [$sitepress, 'rtl_fix']);
+            remove_action('admin_print_styles', [$sitepress, 'rtl_fix']);
+
+            // This caused `switch_lang` to get called. As a result the RTL fix messed up.
+            remove_filter('url_to_postid', [$sitepress, 'url_to_postid']);
+        };
+        $remove_rtl_fix();
+        add_action('wpml_after_startup', $remove_rtl_fix, 10, 0);
+
+        BreakpointsImageSizes::hooks();
```

Comment:
Could we order by asc if are not interrelated? 

I mean
```suggestion
        AuthorPage::hooks();
        BreakpointsImageSizes::hooks();
        QueryLoopPagination::hooks();
        Search\Search::hooks();
        Sendgrid::hooks();
```